### PR TITLE
Publish to pypi

### PR DESCRIPTION
**Pull request by ChuWeiChang**
#4 #8 
I just published the library onto [pypitest](https://test.pypi.org/project/kalgory/).
The email part currently leads to mine, since I lost track of our collective email account on Cloudflare. This is just a publishing test, and official publish will be made after the email section is set correctly and this PR is approved.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+[project]
+name = "kalgory"
+version = "0.0.1"
```
Comment (ZhaoTzuHsien) [inline pyproject.toml:6]: The first version should be 0.1.0

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+authors = [
+  { name="Kalgory-com", email="chuweichang0319@gmail.com" },
```
Comment (ZhaoTzuHsien) [inline pyproject.toml:8]: Set your name in the name field

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+]
```
Comment (ZhaoTzuHsien) [inline pyproject.toml:9]: Add my information after yours

**Review by ZhaoTzuHsien — CHANGES_REQUESTED** [review]
Because the `**kwargs` wasn't supported in `Block.handle()`, please update the usage example.

After you finish the above issue and review  comments, send a new review request to me.

Other codes look nice to me.

**Comment by ZhaoTzuHsien** [comment]
After this PR is merged, don't publish the project by hand since I will setup GitHub Actions to automate this. If you're willing to try and learn setting up GitHub actions, please let me know.

**Comment by ChuWeiChang** [comment]
> After this PR is merged, don't publish the project by hand since I will setup GitHub Actions to automate this. If you're willing to try and learn setting up GitHub actions, please let me know.
You can leave this to me.

**Comment by ChuWeiChang** [comment]
> Because the `**kwargs` wasn't supported in `Block.handle()`, please update the usage example.
Can you be more specific on this? Since we currently use json array, so handle is called by Block.handle(*arg). I don't quite follow.

File: pyproject.toml (hunk 3 above)
Comment (ChuWeiChang) [reply pyproject.toml:9]: Can you give me your email address?
And do you prefer to be called as ZhaoTzu Hsien,  ZhaoTzuHsien ,  Zhao Tzu Hsien or Danny Zhou?

**Comment by ZhaoTzuHsien** [comment]
> > After this PR is merged, don't publish the project by hand since I will setup GitHub Actions to automate this. If you're willing to try and learn setting up GitHub actions, please let me know.
> 
> You can leave this to me.

Here is the docs for GitHub Actions: https://docs.github.com/en/actions

**Comment by ZhaoTzuHsien** [comment]
> > Because the `**kwargs` wasn't supported in `Block.handle()`, please update the usage example.
> 
> Can you be more specific on this? Since we currently use json array, so handle is called by Block.handle(*arg). I don't quite follow.

Since most programming languages don't support keyword arguments, we use JSON array to describe arguments list. Because of that, `*args` should be used instead of `**kwargs`

**Comment by ChuWeiChang** [comment]
> Since most programming languages don't support keyword arguments, we use JSON array to describe arguments list. Because of that, `*args` should be used instead of `**kwargs`

Didn't know you were mentioning README.md. It's updated now.
As for the github actions, I will ask for re-review after they are set up

**Comment by ZhaoTzuHsien** [comment]
> > Since most programming languages don't support keyword arguments, we use JSON array to describe arguments list. Because of that, `*args` should be used instead of `**kwargs`
> 
> Didn't know you were mentioning README.md. It's updated now. As for the github actions, I will ask for re-review after they are set up

You can send review request and I'll check the publish details. For GitHub Actions, create a new issue and PR.

**Review by ZhaoTzuHsien — APPROVED** [review]
LGTM, it can be merged now

**Pull request merged**